app/src/main/java/com/mindorks/framework/mvvm/utils/NetworkHelper.kt

```diff
@@ -23,6 +23,7 @@ class NetworkHelper @Inject constructor(@ApplicationContext private val context:
                 activeNetwork.hasTransport(NetworkCapabilities.TRANSPORT_WIFI) -> true
                 activeNetwork.hasTransport(NetworkCapabilities.TRANSPORT_CELLULAR) -> true
                 activeNetwork.hasTransport(NetworkCapabilities.TRANSPORT_ETHERNET) -> true
+                activeNetwork.hasTransport(NetworkCapabilities.TRANSPORT_VPN) -> true
```

Comment:
Might not so important in sample project. But it's a must in real apps because user expects app to work normal while using VPN.